Components/Avatars/Avatars.css

```diff
@@ -0,0 +1,172 @@
+.avatars {
+  margin: 5rem 2rem;
+}
+
+.avatar {
+  display: flex;
+  gap: 5rem;
+  margin: 3rem;
+  width: 80%;
+  align-items: flex-end;
+}
+
+.radius-50 {
+  border-radius: 50%;
```

Comment:
same with radius like sm , md , lg and xl

Reply:
noted